eConsciencia/src/pages/Quiz/QuizMain.jsx

```diff
@@ -1,22 +1,121 @@
 // src/pages/contamination/BlockWorldMain.jsx
-import React, {useEffect} from 'react';
-import { Canvas, useThree } from '@react-three/fiber';
-import { Environment } from '@react-three/drei';
-import Lights from '@/pages/contamination/lights/Lights'
-import Controls from '@/pages/contamination/controls/Controls';
-import useControlStore from '@/store/use-control-store';
-import QuizBlockWorld from '@/pages/Quiz/QuizBlockWorld';
+import React, { useEffect, useState } from "react";
+import { Canvas, useThree } from "@react-three/fiber";
+import { Environment } from "@react-three/drei";
+import Lights from "@/pages/contamination/lights/Lights";
+import Controls from "@/pages/contamination/controls/Controls";
+import useControlStore from "@/store/use-control-store";
+import QuizBlockWorld from "@/pages/Quiz/QuizBlockWorld";
 
+// estras
+import { db } from "../../../firebase.config";
```

Comment:
The coin system was not assigned to your job, according to what was discussed in the group. 
https://econscience.atlassian.net/browse/SCRUM-101?atlOrigin=eyJpIjoiNjdjMDQ2YTM5NTQxNGVhZTgxM2YzMmFmMWQ5NWUzOTAiLCJwIjoiaiJ9